pymc3/variational/__init__.py

```diff
@@ -1,39 +1,19 @@
 from .advi import advi, sample_vp
 from .advi_minibatch import advi_minibatch
 
-from .updates import (
-    sgd,
-    apply_momentum,
-    momentum,
-    apply_nesterov_momentum,
-    nesterov_momentum,
-    adagrad,
-    rmsprop,
-    adadelta,
-    adam,
-    adamax,
-    norm_constraint,
-    total_norm_constraint,
-)
+# commonly used
+from .updates import *
+from . import updates
 
-from .inference import (
-    ADVI,
-    FullRankADVI,
-    SVGD,
-    fit,
-)
-from .approximations import (
-    Empirical,
-    FullRank,
-    MeanField,
-    sample_approx
-)
+from .inference import *
+from . import inference
 
-from .stein import Stein
+from .approximations import *
```

Comment:
I think we removed all * imports in other parts of the code-base, shouldn't reintroduce it here. cc @ColCarroll